<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OverviewContactsWidget</class>
 <widget class="QDockWidget" name="OverviewContactsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>401</width>
    <height>286</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>280</width>
    <height>279</height>
   </size>
  </property>
  <property name="font">
   <font>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Contact list</string>
  </property>
  <widget class="QWidget" name="mDockWidgetContent">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="leftMargin">
     <number>3</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>3</number>
    </property>
    <property name="bottomMargin">
     <number>3</number>
    </property>
    <item>
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::Panel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="margin">
        <number>3</number>
       </property>
       <item>
        <widget class="QLabel" name="mLbContactHint">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>60</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
           <italic>true</italic>
          </font>
         </property>
         <property name="frameShape">
          <enum>QFrame::Panel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <property name="text">
          <string>Right click here to add a new contact, &lt;br&gt;afterwards right click on the new contact</string>
         </property>
         <property name="scaledContents">
          <bool>false</bool>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTreeView" name="mTvContacts">
         <property name="contextMenuPolicy">
          <enum>Qt::CustomContextMenu</enum>
         </property>
         <property name="frameShape">
          <enum>QFrame::Panel</enum>
         </property>
         <property name="tabKeyNavigation">
          <bool>true</bool>
         </property>
         <property name="dragDropMode">
          <enum>QAbstractItemView::DropOnly</enum>
         </property>
         <property name="textElideMode">
          <enum>Qt::ElideLeft</enum>
         </property>
         <property name="rootIsDecorated">
          <bool>false</bool>
         </property>
         <property name="uniformRowHeights">
          <bool>true</bool>
         </property>
         <property name="sortingEnabled">
          <bool>true</bool>
         </property>
         <property name="animated">
          <bool>true</bool>
         </property>
         <attribute name="headerVisible">
          <bool>true</bool>
         </attribute>
         <attribute name="headerCascadingSectionResizes">
          <bool>true</bool>
         </attribute>
         <attribute name="headerDefaultSectionSize">
          <number>30</number>
         </attribute>
         <attribute name="headerMinimumSectionSize">
          <number>30</number>
         </attribute>
         <attribute name="headerShowSortIndicator" stdset="0">
          <bool>true</bool>
         </attribute>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QToolButton" name="mTbSaveList">
           <property name="minimumSize">
            <size>
             <width>27</width>
             <height>27</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>27</width>
             <height>27</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Save contact list</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../resources/Icons.qrc">
             <normaloff>:/images/DriveSave.png</normaloff>:/images/DriveSave.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="mTbLoadList">
           <property name="minimumSize">
            <size>
             <width>27</width>
             <height>27</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>27</width>
             <height>27</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Load contact list</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../resources/Icons.qrc">
             <normaloff>:/images/BoxOpen.png</normaloff>:/images/BoxOpen.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="mTbAdd">
           <property name="minimumSize">
            <size>
             <width>27</width>
             <height>27</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>27</width>
             <height>27</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Add user to contact list</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../resources/Icons.qrc">
             <normaloff>:/images/Plus.png</normaloff>:/images/Plus.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>18</width>
             <height>18</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="mTbDel">
           <property name="minimumSize">
            <size>
             <width>27</width>
             <height>27</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>27</width>
             <height>27</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Remove user from contact list</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../resources/Icons.qrc">
             <normaloff>:/images/Minus.png</normaloff>:/images/Minus.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>18</width>
             <height>18</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::MinimumExpanding</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>5</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../resources/Icons.qrc"/>
 </resources>
 <connections/>
</ui>
